### PR TITLE
Add a label for "expenditure-type".

### PR DESCRIPTION
**Pull request by cyberbikepunk**
Should fix issue #1074.

### CODE REVIEW

**Comment by cyberbikepunk** [comment]
okay the first commit doesn't pass the test. I'm fixing this.

**Comment by cyberbikepunk** [comment]
@akariv PR seems to be ready for a merge.

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,41 @@
+0 info it worked if it ends with ok
```
Comment (akariv) [inline npm-debug.log]: You mistakenly committed this file...

**Comment by akariv** [comment]
Other than one mistakenly committed file, looks good.

**Comment by akariv** [comment]
@cyberbikepunk this is waiting for you...

**Pull request merged**